src/constructs/iam/policies/anghammarad.ts

```diff
@@ -1,30 +1,30 @@
 import { isSingletonPresentInStack } from "../../../utils/test";
 import type { GuStack } from "../../core";
-import { AnghammaradTopicParameter } from "../../core/parameters/anghammarad";
+import { GuAnghammaradTopicParameter } from "../../core";
 import { GuAllowPolicy } from "./base-policy";
 
 /**
  * Creates an `AWS::IAM::Policy` to grant `sns:Publish` permission to the Anghammarad topic.
  * An `AnghammaradSnsArn` parameter will be automatically added to the stack when needed.
  *
- * @see AnghammaradTopicParameter
+ * @see GuAnghammaradTopicParameter
  * @see https://github.com/guardian/anghammarad
  */
-export class AnghammaradSenderPolicy extends GuAllowPolicy {
-  private static instance: AnghammaradSenderPolicy | undefined;
+export class GuAnghammaradSenderPolicy extends GuAllowPolicy {
+  private static instance: GuAnghammaradSenderPolicy | undefined;
 
   private constructor(scope: GuStack) {
-    const anghammaradTopicParameter = AnghammaradTopicParameter.getInstance(scope);
+    const anghammaradTopicParameter = GuAnghammaradTopicParameter.getInstance(scope);
 
-    super(scope, "GuSESSenderPolicy", {
```

Comment:
I didn't notice this in the last PR review 😅 Thanks for fixing!

Reply:
You know know my secret 😊 .